recipes/pango/all/conanfile.py

```diff
@@ -13,8 +13,8 @@ class PangoConan(ConanFile):
     homepage = "https://www.pango.org/"
     topics = ("conan", "fontconfig", "fonts", "freedesktop")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "libthai": [True, False], "cairo": [True, False], "xft": [True, False, "auto"], "freetype": [True, False], "fontconfig": [True, False, "auto"]}
+    default_options = {"shared": False, "fPIC": True, "libthai": False, "cairo": True, "xft": "auto", "freetype": True, "fontconfig": "auto"}
```

Comment:
please follow the convention of `with_dep` for the options

Reply:
Done in 541fe4e872319dbbb0b204b3f81cda03651d656c